pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -491,17 +510,17 @@ def _validate_monitor_key(self, trainer):
 
     def _get_metric_interpolated_filepath_name(
         self,
-        ckpt_name_metrics: Dict[str, Any],
+        monitor_candidates: Dict[str, Any],
```

Comment:
what is the candidates meaning?

Reply:
It is a dictionary to match what can be monitored to each value.

Here i'm just renaming for consistency since we use `monitor_candidates` in the rest of the file and it also sounds better to me.